server/routes/github/github_user_repos.py

```
@@ -30,148 +85,395 @@ def create_cors_response(data=None, status=200):
+                repos_by_full_name[full_name] = _simplify_repo(
+                    repo, "app", installation_id,
+                )
+
+    # OAuth fallback / additional source. App entries already loaded above
+    # win on collision per the module docstring (finer permissions, isolated
+    # rate limits). Only runs when OAuth is enabled in the deployment AND
+    # the user has a stored token.
+    if is_oauth_enabled():
+        try:
+            creds = get_credentials_from_db(user_id, "github")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)